electricpy/__init__.py

```diff
@@ -4269,28 +4426,61 @@ def secondary(val, Np, Ns=1,invert=False):
                 used when reflecting current across a
                 voltage transformer, or voltage across a
                 current transformer.
-    
+
     Returns
     -------
     reflection: complex
                 The reflected value referred to the secondary
                 side according to Np and Ns.
     """
     if invert:
-        return( val * Np/Ns )
-    return( val * Ns/Np )
+        return (val * Np / Ns)
+    return (val * Ns / Np)
+
+
+def tap_changing_transformer(v1, v2, p, q, r, x):
+    '''
+        calulating turn ration for on load tap chaining transformer
```

Comment:
I think we should probably elaborate a little further on what the purpose of this function is, what it does, and why it might be useful. We should probably also consider adding a formula to help users understand what the formula does under the hood (mathematically speaking).

Reply:
updated in subsequent PR

Reply:
may decide to further elaborate on this in the future, but for the time being, the description is sufficient